### PR TITLE
fix: Log exception's message with stacktrace

### PR DESCRIPTION
**Pull request by lolgab**

### CODE REVIEW

**Comment by mrfyda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/main/scala/codacy/plugins/test/MetricsTests.scala  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/codacy-plugins-test/codacy-plugins-test/pullRequest?prid=4567470&bid=15356297)

File: src/main/scala/codacy/plugins/test/PluginsTests.scala [hunk 1]
```diff
@@ -74,7 +75,7 @@ object PluginsTests extends ITest {
             true
           }
         case Failure(exception) =>
```
Comment (peavers) [inline src/main/scala/codacy/plugins/test/PluginsTests.scala]: Can we be constant on the naming of exceptions? Either as `e` or `exception`

Comment (lolgab) [reply src/main/scala/codacy/plugins/test/PluginsTests.scala]: Sure :)

Comment (lolgab) [reply src/main/scala/codacy/plugins/test/PluginsTests.scala]: Done 👍

**Comment by mrfyda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/main/scala/codacy/plugins/test/MetricsTests.scala  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/codacy-plugins-test/codacy-plugins-test/pullRequest?prid=4567470&bid=15356297)

**Pull request merged**